llama_index/param_tuner/base.py

```diff
@@ -262,13 +262,16 @@ def param_fn_wrapper(
 
         results = tuner.fit()
         all_run_results = []
-        for result in results:
+        for idx in range(len(results)):
+            result = results[idx]
```

Comment:
what's motivation of this change?

Reply:
oh lol. this was causing a lint error if you install ray tune (ResultGrid is not iterable but it has getitem implemented..)

Reply:
lol gotcha!